backend/src-common/src/main/java/com/siemens/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -247,7 +244,8 @@ public Project getProjectByIdforEdit(String id, User user) throws SW360Exception
         } else {
             final String email = user.getEmail();
             Optional<ModerationRequest> moderationRequestOptional = CommonUtils.getFirstModerationRequestOfUser(moderationRequestsForDocumentId, email);
-            if (moderationRequestOptional.isPresent()) {
+            if (moderationRequestOptional.isPresent()
+                    && (moderationRequestOptional.get().getModerationState().equals(ModerationState.INPROGRESS)|| moderationRequestOptional.get().getModerationState().equals(ModerationState.PENDING))) {
```

Comment:
Since this us used frequently, you could encapsulate this into a function call `moderationRequestOptional.get().isInprogressOrPending()`